test/e2e/basic_scheduling/basic_scheduling_test.go

```diff
@@ -70,6 +70,16 @@ var _ = ginkgo.Describe("", func() {
 		Ω(err3).NotTo(HaveOccurred())
 		Ω(d).NotTo(BeNil())
 
+		ginkgo.By("Restart scheduler pod")
+		_, err4 := kClient.ScaleDeployment(configmanager.YKScheduler, 0, configmanager.YuniKornTestConfig.YkNamespace)
```

Comment:
Note k8s has no API call to bounce a pod. I'm simply scaling the replicas in the scheduler deployment to 0 and then back to 1.